src/mixxx.cpp

```diff
@@ -1456,24 +1448,8 @@ void MixxxMainWindow::rebootMixxxView() {
 
     if (wasFullScreen) {
         slotViewFullScreen(true);
-    } else if (!initSize.isEmpty()) {
-        // Not all OSs and/or window managers keep the window inside of the screen, so force it.
-        int newX = initPosition.x() + (initSize.width() - m_pCentralWidget->width()) / 2;
-        int newY = initPosition.y() + (initSize.height() - m_pCentralWidget->height()) / 2;
-
-        const QScreen* const pScreen = mixxx::widgethelper::getScreen(*this);
-        VERIFY_OR_DEBUG_ASSERT(pScreen) {
-            qWarning() << "Unable to move window inside screen borders.";
-        }
-        else {
-            const auto windowMarginWidth =
-                    pScreen->geometry().width() - m_pCentralWidget->width();
-            const auto windowMarginHeight =
-                    pScreen->geometry().height() - m_pCentralWidget->height();
-            newX = std::max(0, std::min(newX, windowMarginWidth));
-            newY = std::max(0, std::min(newY, windowMarginHeight));
-            move(newX, newY);
-        }
+    } else {
+        setGeometry(initGeometry);
```

Comment:
Please add a comment. About what will happen if the new skin minimum is bigger than the initGeometry.